yarn.lock

```diff
@@ -3208,6 +3208,7 @@ __metadata:
     "@storybook/test": "npm:^8.3.5"
```

Comment:
Updates to yarn.lock because of updated `@ts-bridge/cli` dependencies in design-tokens, design-system-tailwind-preset and design-system-react